event_sink_clickhouse/sinks/course_published.py

```diff
@@ -219,7 +219,3 @@ def get_course_last_published(self, course_key):
 
     def convert_id(self, item_id):
         return CourseKey.from_string(item_id)
-
-    def get_queryset(self):
```

Comment:
There were no differences from this change?

Reply:
This change allow to include `ccx` courses in the sink.

Reply:
But the output remains the same / this still works with the existing CH schema?

Reply:
Yes, the output remains the same.